DIPs/DIP1016.md

```diff
@@ -175,6 +175,32 @@ Again where initial construction of `result` should be performed at the moment o
 
 It is important that `T` be defined as the _parameter_ type, and not `auto` (ie, the argument type), because it will allow implicit conversions to occur naturally, with identical behavior as when the parameter is not `ref`. The user should not experience edge cases, or differences in functionality when calling `fun(int x)` vs `fun(ref int x)`.
 
+### `@property` and implicit wideing casts
+
+`@property` and implicit wideing casts are syntactically identical to public feilds and variables respectively and so a change in the API
+from one to the other would result in silent change to the users code. 
```

Comment:
result in silent change to the users code -> result in a silent change in users' code

Although, I think what you're more accurately referring to here is the behavior of user code, correct? In which case "a silent change in the behavior of users' code". Personally, I would prefer the compound noun of "user code" to the plural possessive form (those irk me for some reason), but it's a matter of preference.